test/inductor/test_aot_inductor_arrayref.py

```diff
@@ -78,6 +78,8 @@ def fail_minimal_arrayref_interface(is_skip=False):
     "test_while_loop_with_mixed_device_dynamic_False": fail_stack_allocation(),
     "test_while_loop_with_sym_expr_cond_dynamic_True": fail_minimal_arrayref_interface(),
     "test_while_loop_with_sym_expr_cond_dynamic_False": fail_minimal_arrayref_interface(),
+    "test_while_loop_with_conv_dynamic_True": fail_minimal_arrayref_interface(),
+    "test_while_loop_with_conv_dynamic_False": fail_minimal_arrayref_interface(),
```

Comment:
what is this?

Reply:
There are some aot_inductor infra don't support codegen hops I think. This follows how we did it previously.